README.md

```diff
@@ -52,8 +51,7 @@ vastParser.parseVAST(vastXml, (err, res) => {
   })
   .catch(err => {
     // Deal with the error
-  })
-});
+  });
```

Comment:
This is actually not good enough, since the method call still shows some syntax from the old callback-based interface.

So the line `48`
```javascript
vastParser.parseVAST(vastXml, (err, res) => {
```
Should become
```javascript
vastParser.parseVAST(vastXml)
```

Reply:
@eliamaino-fp fixed, please CR again.